dockerize.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
i think we should also update the other dockerfiles then? so they're consistent?